src/azure-cli/azure/cli/command_modules/identity/commands.py

```diff
@@ -18,10 +18,10 @@ def load_command_table(self, _):
         client_factory=_msi_user_identities_operations
     )
 
-    with self.command_group('identity', identity_sdk) as g:
-        g.command('create', 'create_or_update', validator=process_msi_namespace)
+    with self.command_group('identity', identity_sdk, client_factory=_msi_user_identities_operations) as g:
+        g.custom_command('create', 'create_identity', validator=process_msi_namespace)
         g.show_command('show', 'get')
         g.command('delete', 'delete')
         g.custom_command('list', 'list_user_assigned_identities')
-        g.command('list-operations', 'list', operations_tmpl='azure.mgmt.msi.operations.operations#Operations.{}',
+        g.command('list-operations', 'list', operations_tmpl='azure.mgmt.msi.operations#Operations.{}',
```

Comment:
Due to the existence of non customized commands (such as `delete` and `list-operations`) and their operation path is hard coded as `azure.mgmt.msi.operations#xxx`. 
If `ResourceType.MGMT_MSI: '2018-11-30'` configuration is not added to the old profiles, CLI will report an error when it executes a non customized command in the old profile which without adding ``ResourceType.MGMT_MSI`.
![Screenshot 2022-03-30 211508](https://user-images.githubusercontent.com/16612065/160845984-2fcc30f1-57ab-4c34-91f0-ad05820298d2.png)

This is because when loading arguments, a imports the operation path as hard coded path, which will cause the package from operation path cannot be found because the SDK package is a multiple version package.
code link: https://github.com/Azure/azure-cli/blob/45842acafb076d58f66511c46192e039e3f815f0/src/azure-cli-core/azure/cli/core/commands/command_operation.py#L41-L59
![Screenshot 2022-03-30 211424](https://user-images.githubusercontent.com/16612065/160846269-49426dae-e0bc-4c10-9e93-64db21a741ad.png)

When we add the configuration `ResourceType.MGMT_MSI` in the profile,  the import operation path is the correct path containing the version number in the multiple version SKD
![Screenshot 2022-03-30 210707](https://user-images.githubusercontent.com/16612065/160846514-75a8d3dd-41ad-4ce3-9b62-142c2a6e7a9a.png)

Reply:
The relevant processing logic of import operation path: https://github.com/Azure/azure-cli/blob/45842acafb076d58f66511c46192e039e3f815f0/src/azure-cli-core/azure/cli/core/commands/command_operation.py#L41-L59

Reply:
We can also add `resource_type=ResourceType.MGMT_MSI` to the `__init__()`, so that the identity module like many other modules which are only supported in the latest profile.
```python
   def __init__(self, cli_ctx=None):
        from azure.cli.core.commands import CliCommandType
        from azure.cli.core.profiles import ResourceType
        identity_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.identity.custom#{}')
        super(IdentityCommandsLoader, self).__init__(cli_ctx=cli_ctx,
                                                     resource_type=ResourceType.MGMT_MSI,
                                                     custom_command_type=identity_custom)
```

However, when the identity module originally used the Python SDK with single version, it can support all the profiles. Therefore, such modification may cause unnecessary breaking changes in the old profiles. 
So we add the `ResourceType.MGMT_MSI` to all profiles to still make `identity` module can support all profiles